src/utils/resources/template/utils/helpers.ts

```diff
@@ -33,3 +39,23 @@ export const replaceTemplateVM = (template: V1Template, vm: V1VirtualMachine) =>
     draftTemplate.objects.splice(vmIndex, 1, vm);
   });
 };
+
+export const createAllTemplateObjects = (
+  template: V1Template,
+  models: { [key: string]: K8sModel },
+  namespace?: string,
+): Promise<K8sResourceCommon[]> =>
+  Promise.all(
+    template.objects.map((object) => {
+      const { group, version, kind } = getGroupVersionKindForResource(object);
+
+      const ref = [group || 'core', version, kind].join('~');
```

Comment:
you can use `transformGroupVersionKindToReference` here

Reply:
It's deprecated. Can I use it anyway?

Reply:
Can't import this function...